### PR TITLE
Gradle IT: if set, pass maven.repo.local to the run task

### PR DESCRIPTION
**Pull request by aloubyansky**
In case the local maven repo specified with maven.repo.local system property, it should be propagated to the run task explicitly in the config.

### CODE REVIEW

**Review by glefloch — APPROVED** [review]
Oh nice catch ! LGTM

**Comment by aloubyansky** [comment]
The merge button does not appear to be enabled for me even though the PR has been approved. @gsmet is that a new feature or something weird?

**Comment by gsmet** [comment]
You need the approval of a committer.

**Review by gsmet — APPROVED** [review]

**Pull request merged**
